ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/common-settings.md

```diff
@@ -237,6 +238,7 @@ platform_settings:
     reload_delay:
       mlag: 300
       non_mlag: 330
+    queue_monitor_length_notify_support: false
```

Comment:
In order to support more feature-specific knobs, let's update to the following!

```yaml
feature_support:
  queue_monitor_length_notify:
```

Reply:
Done.